randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -192,15 +188,9 @@ Extra - asset_id: collision_camera_003
 > Center Platform; Heals? False
   * Layers: default
   > Dock to Teleport to Burenia (Cyan)
-      Any of the following:
-          Gravity Suit or Use Spin Boost
-          All of the following:
-              Flash Shift
-              Grapple Beam or Spider Magnet
+      Grapple Beam or Gravity Suit or Use Spin Boost
   > Door to EMMI Zone Exit Middle (Charge)
-      Any of the following:
-          Grapple Beam or Gravity Suit or Use Spin Boost
-          Flash Shift and Spider Magnet
+      Grapple Beam or Gravity Suit or Use Spin Boost
```

Comment:
Looks like this room is the only issue left. Both these connections need to have Grapple Beam with Grapple Movement. Also, is Magnet alone enough? Looks like you can jump straight to the ones on the wall from the center. If not, it would be Grapple and (Grapple Movement or Magnet). Still need to add the trivial connection to the Power Beam door to EMMI Zone Exit Middle too, and then remove the connection from the Charge door to here.